docs/Troubleshooting.md

```diff
@@ -61,3 +61,23 @@ sudo chown -R $USER /usr/local/lib/node_modules
 
 ## Debugging in Chrome hangs and/or does not work well
 It is possible that one of your Chrome extensions is interacting in unexpected ways with the debugger. If you are having this issue, try disabling all of your extensions and re-enabling them one-by-one until you find the problematic extension.
+
+## Xcode Build Failures
+
+To see the exact error what is causing your build to fail, go into the Issues Navigator in the left sidebar.
+
+##### React libraries missing
+If you are using CocoaPods, verify that you have added React along with the subspecs to the `Podfile`. For example, if you were using the `<Text />`, `<Image />` and `fetch()` APIs, you would need to add these in your `Podfile`:
+```
+pod 'React'
+pod 'React/RCTText'
+pod 'React/RCTImage'
+pod 'React/RCTNetwork'
+```
```

Comment:
Subspecs array syntax might be easier to work with since RN has a ton of subspecs. Also might be worth including the remote debugger and RCTWebSocket in the example to head off questions about why it's not working.

```ruby
pod 'React', :subspecs => [
  'RCTText',
  'RCTImage',
  'RCTNetwork',
  'RCTWebSocket',
]
```